api/lang/server.go

```diff
@@ -145,8 +148,10 @@ func (h *LangHandler) Handle(ctx context.Context, conn jsonrpc2.JSONRPC2, req *j
 		return getCodeActions(req)
 	case "gauge/scenarios":
 		return getScenarios(req)
+	case "gauge/specs":
+		return getSpecs()
 	default:
-		return nil, nil
+		return nil, fmt.Errorf("Unknown request : %s" + req.Method)
```

Comment:
vscode sends a lot of requests like didSave, which we don't make use of. Just because we are not handling these requests, we shouldn't send an error. Also, any errors sent to the client gets printed on the client gauge output panel. It will lead to a lot of noise.